ansible/roles/analytics-bootstrap-spark/tasks/main.yml

```diff
@@ -10,6 +10,36 @@
   with_items:
     - {var: 'azure_storage_key', value: '{{ sunbird_private_storage_account_name }}'}
     - {var: 'azure_storage_secret', value: '{{ sunbird_private_storage_account_key }}'}
+ 
+
+- name: Adding PATH for oci cli Vars to bashrc file of spark.
+  become: yes
+  become_user: "{{ analytics_user }}"
+  lineinfile:
+    path: '{{ analytics_user_home }}/.bashrc'
+    line: 'export PATH={{ analytics_user_home }}/bin:$PATH'
```

Comment:
Does this script provision the 3.2.1 version of Spark? @ddevadat

Reply:
analytics-bootstrap-spark role , doesnt actually installs spark. Its just creates the necessary files and folder structure.
the provisioning is done by  the role analytics-spark-provision.

The default spark version by the role analytics-spark-provision is still at 3.1.3, but i think it can be overridden by the ansible variable spark_version. Or the other option could be to change the default to 3.2.1

Reply:
Yes. I am aware of it. The analytics-spark-provision role is responsible for installing the spark of a specific version.
I would suggest updating the default values of the spark provision ansible role instead of pointing to the older version

Here - https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/blob/release-5.1.1/ansible/roles/analytics-spark-provision/defaults/main.yml#L6

Reply:
i updated the provisioning version to 3.2.1